main.go

```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"github.com/hashicorp/terraform/plugin"
 	"github.com/hashicorp/terraform/terraform"
-	"terraform-provider-esxi/esxi"
+	"github.com/terraform-providers/terraform-provider-esxi/esxi"
```

Comment:
Why is this not using `josenk` repo?

Reply:
Good question, I've made changes according to this comment: https://github.com/josenk/terraform-provider-esxi/issues/32
It works for me this way, maybe it's a trick to include this package as terraform plugin.